beetsplug/bpd/__init__.py

```diff
@@ -939,11 +1018,19 @@ def cmd_status(self, conn):
         if self.current_index > -1:
             item = self.playlist[self.current_index]
 
-            yield u'bitrate: ' + six.text_type(item.bitrate / 1000)
-            # Missing 'audio'.
+            yield (
+                u'bitrate: ' + six.text_type(item.bitrate / 1000),
+                # TODO provide a real value samplerate:bits:channels 44100:24:2
+                u'audio: 0:0:0',
```

Comment:
FWIW, this should actually be possible to provide: `item.samplerate`, `item.bitdepth`, and `item.channels` are all standard fields.

Reply:
Thanks, I was assuming this had to come from the player and didn't realise beets already knows all this!